single-document.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * The template used for displaying Publication Pages
+ *
+ * @package phila-gov
+ */
+
+get_header();
+?>
+<?php /*
+
+Allow screen readers / text browsers to skip the navigation menu and
+get right to the good stuff. */ ?>
+
+<div class="skip-link screen-reader-text">
+		<a href="#content" title="<?php esc_attr_e( 'Skip to content', 'twentyten' ); ?>">
+		<?php _e( 'Skip to content', 'twentyten' ); ?></a>
+</div>
+
+<article id="post-<?php the_ID(); ?>" <?php post_class('row document'); ?>>
+<?php
+  	//loop for our regularly scheduled content
```

Comment:
I prefer one tab. Some of this is due to editor switching and little time to go back for clean up.